src/OpenLoco/src/Input/Shortcuts.cpp

```diff
@@ -28,19 +30,28 @@ namespace OpenLoco::Input::Shortcuts
         WindowManager::closeTopmost();
     }
 
-    // 004BF0B6
+    // 0x004BF0B6
     static void closeAllFloatingWindows()
     {
         WindowManager::closeAllFloatingWindows();
     }
 
-    // 0x4BF0BC
+    // 0x004BF0BC
     static void cancelConstructionMode()
     {
-        call(0x004BF0BC);
+        if (auto* w = WindowManager::find(WindowType::error))
```

Comment:
in general we don't implicitly do nullptr compares. Can you refactor this to have a != nullptr

Reply:
You mean like this?
```suggestion
        if ((auto* w = WindowManager::find(WindowType::error)) && w != nullptr)
```

Reply:
Or just two line it

Reply:
Sure, I'll go with this then:
```suggestion
        auto* w = WindowManager::find(WindowType::error);
        if (w != nullptr)
```